src/common_c.py

```diff
@@ -16,7 +17,17 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with libocispec.  If not, see <http://www.gnu.org/licenses/>.
-#!/usr/bin/python -Es
```

Comment:
Why delete it? Is a mistake?

Reply:
AFAIK, it is not used if it is not the first line in the file

Reply:
ok, got it. LGTM.